react/__tests__/AddressForm.test.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { render, fireEvent } from '@vtex/test-tools/react'
+import { AddressContextProvider } from 'vtex.address-context/AddressContext'
+import { Address } from 'vtex.checkout-graphql'
+import AddressForm from '../AddressForm'
+import { sampleAddress as address } from '../__mocks__/mockDescriptions'
+
+describe('Address Form', () => {
+  const renderComponent = (address: Address) => {
+    return render(
+      <AddressContextProvider address={address}>
+        <AddressForm />
+      </AddressContextProvider>
+    )
+  }
+
+  it('should render already filled information as read-only', () => {
+    const { queryByDisplayValue, getByText } = renderComponent(address)
+
+    expect(getByText(address.street as string)).toBeTruthy()
+    expect(getByText(address.number as string)).toBeTruthy()
+    expect(getByText(address.neighborhood as string)).toBeTruthy()
+    expect(getByText(address.city as string)).toBeTruthy()
+    expect(getByText(address.state as string)).toBeTruthy()
+    expect(getByText(address.postalCode as string)).toBeTruthy()
+
+    expect(queryByDisplayValue(address.street as string)).toBeFalsy()
+    expect(queryByDisplayValue(address.number as string)).toBeFalsy()
+    expect(queryByDisplayValue(address.neighborhood as string)).toBeFalsy()
+    expect(queryByDisplayValue(address.city as string)).toBeFalsy()
+    expect(queryByDisplayValue(address.state as string)).toBeFalsy()
+    expect(queryByDisplayValue(address.postalCode as string)).toBeFalsy()
+  })
```

Comment:
the test name says it checks the inputs to be `read-only`, but there's nothing asserting that in the function. are you still working on these tests?